Cargo.toml

```diff
@@ -29,5 +29,5 @@ path = "src/lib.rs"
 crate-type = [ "dylib" ]
 plugin = true
 
-[dev_dependencies.gfx]
-git = "https://github.com/gfx-rs/gfx-rs"
+[dev_dependencies]
+gfx = "*"
```

Comment:
That would be good, yeah. I still don't quite understand though. If you don't re-publish gfx after every dependent change, your copy will still compile (due to cargo overrides), but everyone else will be broken. So is it really good to override?